src/EditorFeatures/Core/Extensibility/NavigationBar/NavigationBarItems/RoslynNavigationBarItem.GenerateEventHandlerItem.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Editor.Extensibility.NavigationBar
+{
+    internal abstract partial class RoslynNavigationBarItem
+    {
+        public class GenerateEventHandler : AbstractGenerateCodeItem
```

Comment:
Why did these have to move out of the VB code? My "why" question here isn't that this is _bad_ just not sure where you're going with this.